scripts/email_sample_disposal_notification.py

```diff
@@ -7,8 +7,6 @@
 class SampleDisposalNotificationEmail(SendMailEPP):
 
     def _run(self):
-        if len(self.projects) > 1:  # check if more than one project in step, only one permitted
-            raise ValueError('More than one project present in step. Only one project per step permitted')
 
         # Create the message
         msg = 'Hi,\n\nThe samples at the link below have been approved for disposal by the Facility Manager:\n' \
```

Comment:
`sample_count` and `project` below are no longer used in this format string, and can be removed.